.github/workflows/regression_test_full.yml

```diff
@@ -0,0 +1,41 @@
+name: Run Regression Full Test
+
+on: 
+  workflow_call:
+
+jobs: 
+  test:
+    name: Regression Test - Full
+    runs-on: ubuntu-latest
+    steps: 
```

Comment:
I set up [an environment for running on ACCESS](https://github.com/FCP-INDI/C-PAC/settings/environments/1184714702/edit)
![ACCESS environment protection rules](https://github.com/FCP-INDI/C-PAC/assets/5974438/81c14ab5-a9b2-4b71-a7a9-f5e0e1f5fb5c), and we can apply it by including `environment: ACCESS` in any jobs where we want these rules to apply like
```suggestion
    runs-on: ubuntu-latest
    environment: ACCESS
    steps: 
```


I set "Allow administrators to bypass configured protection rules" to true; that's what we want, right?